browser-test/src/admin_program_viewing.test.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  createTestContext,
+  enableFeatureFlag,
+  loginAsAdmin,
+  validateScreenshot,
+} from './support'
+
+describe('admin program view page', () => {
+  const ctx = createTestContext()
+
+  it('view active program, without draft and after creating draft', async () => {
```

Comment:
**suggestion (blocking):** please add some questions to the program

Reply:
Done.

Reply:
It looks like this test case still doesn't have a question added to the program.

Reply:
Ah got it, sounds good